tests/gordo_components/model/test_model.py

```diff
@@ -203,62 +222,35 @@ def test_lookback_window_forecast_valueerror_during_transform(self):
             with self.assertRaises(ValueError):
                 model.fit(X)
 
-    def test_generate_window_lstm_ae_output(self):
-        # Check that right output is generated from generate_window (for KerasLSTMAutoEncoder)
+    def test_get_TimeseriesGenerator(self):
```

Comment:
Are the lookahead (and less importantly batch_size) features properly tested here? Seems like all tests have lookahead=1 and batch_size=1. I'm not saying that we should test the TimeseriesGenerator, but perhaps the shifting we do with the lookahead should be tested a bit more?

Reply:
it should, I updated now.